pallets/compliance-manager/src/lib.rs

```diff
@@ -257,146 +261,152 @@ decl_module! {
         fn deposit_event() = default;
 
         fn on_runtime_upgrade() -> frame_support::weights::Weight {
+            use frame_support::migration::{StorageIterator, put_storage_value};
             use polymesh_primitives::migrate::migrate_map;
-            migrate_map::<AssetTransferRulesOld>(b"ComplianceManager", b"AssetRulesMap");
 
-            // It's gonna be alot, so lets pretend its 0 anyways.
-            0
+            // ComplianceManager.AssetRulesMap -> ComplianceManager.AssetCompliance
```

Comment:
This seems right, but I'd prefer `migrate_map_rename` (base function used to impl `migrate_map`) below (using 2 pairs perhaps) to do this in one shot.